app/views/RoomView/index.js

```diff
@@ -313,9 +337,70 @@ export default class RoomView extends React.Component {
 		}
 	}
 
-	onMessageLongPress = (message) => {
-		const { actionsShow } = this.props;
-		actionsShow({ ...message, rid: this.rid });
+	onMessageLongPress = (item) => {
```

Comment:
Too many lines added from one file to another.
You're adding too much complexity and turning this file harder to read.
Also this is going to impact performance, since you've added several binds that aren't going to be used every time.
Can you do it in another one file?